src/components/ProfilePage/Certificates.jsx

```diff
@@ -91,6 +91,14 @@ class Certificates extends React.Component {
             </p>
             <p className="h6 mb-4">{courseOrganization}</p>
             <div className="flex-grow-1" />
+            <p className="small mb-2">
+              <FormattedMessage
+                id="profile.certificate.completion.date.label"
+                defaultMessage="Completed on"
```

Comment:
I think you want something like: `"Completed on {date}"`.  See [this example from footer](https://github.com/edx/frontend-component-footer/blob/66f6832887f3cf34b1a32e929a1adfda862c4a49/src/lib/components/SiteFooter/index.jsx#L276-L281).

Reply:
It should probably also use FormattedDate. Maybe something like this?
```
<FormattedMessage
  id="profile.certificate.completion.date.label"
  defaultMessage="Completed on {date}"
  values={{
    date: <FormattedDate value={new Date(modifiedDate)}/>,
  }}
/>
```
FormattedDate Reference: https://github.com/yahoo/react-intl/wiki/Components#formatteddate

Reply:
Thanks! Pushing update now.